### PR TITLE
Fixed autocomplete bug

### PR DESCRIPTION
**Pull request by aniket-teltumbade-au9**

### CODE REVIEW

File: react-hook-form-mui/src/Input.js [hunk 1]
```diff
@@ -27,36 +27,34 @@ ErrorBoundary.propTypes = {
...
-	}
-	return rhf ?  require('./react-hook-form/TextField').default : require('./forms/TextField').default;
-};
-
-
 // eslint-disable-next-line react/no-multi-comp
 function Input({type, container, validate, label, rhf = true, components: {input, Field = input, Loader = LinearProgress, ...components} = {}, compact, ...rest}) {// eslint-disable-lineno-unused-varsno-unused-vars
 	const Container = container ? require('@mui/material/Grid').default : Fragment;
 	const containerProps = container ? {item: true, ...container} : {};
+	const module = ({type, rhf}) => {
```
Comment (s7dhansh) [inline react-hook-form-mui/src/Input.js:34]: why is it inside input?

Comment (aniket-teltumbade-au9) [reply react-hook-form-mui/src/Input.js:34]: Sir, It wasn't detecting Input component inside default export. It was returning "undefined".

Comment (aniket-teltumbade-au9) [reply react-hook-form-mui/src/Input.js]: Sorry Sir, I tested it again and removed it from PR.

**Comment by s7dhansh** [comment]
please send all prs to dev - not to master. please rebase

**Pull request merged**